Gems/Terrain/Code/Source/TerrainRenderer/TerrainMeshManager.cpp

```diff
@@ -149,36 +150,41 @@ namespace Terrain
         terrainSrg->SetConstant(m_srgMeshDataIndex, meshData);
     }
 
-    void TerrainMeshManager::CheckStacksForUpdate(AZ::Vector3 newPosition)
+    void TerrainMeshManager::CheckLodGridsForUpdate(AZ::Vector3 newPosition)
     {
-        AZStd::vector<AZStd::vector<StackSectorData*>> sectorsToUpdate;
-        sectorsToUpdate.resize(m_sectorStack.size());
+        // lots of sectors that need updating, separated by LOD level.
+        AZStd::vector<AZStd::vector<Sector*>> sectorsToUpdate;
+        sectorsToUpdate.resize(m_sectorLods.size());
```

Comment:
Nit: could condense this to
```
AZStd::vector<AZStd::vector<Sector*>> sectorsToUpdate(m_sectorLods.size());
```

Also, FWIW, I interpreted "lots of sectors" as "bunches of sectors". Maybe "groups of sectors" or "lists of sectors"?

Reply:
lol, it's supposed to be Lods 😄 will fix both.